app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="parking_button_margin_top">20dp</dimen>
```

Comment:
this should be a more specific name like "activity_main_button_margin_top"

Reply:
Done.